### DISCUSSION TITLE
PHPMailer not working after Update

### DISCUSSION THREAD

**Discussion by thepersonthatexists** [PHPMailer/PHPMailer#2442] [Q&A]
I recently updated to the latest version of PHP and PHPMailer, and used the same credentials as before. Nothing has changed and I am still using SMTP, and the emails sending seems to be working. It returns no errors, even with the highest level error reporting, and I am seeing the same results when I try to send emails locally using Live Server and on my website. Does anyone have possible solutions to emails not being sent even though the script thinks it's successful?

**Comment by Synchro** [comment]
There is a big difference between sending email and receiving it. Without code or debug output we can't give you any pointers. You'll need to look at your mail server's log file to tell what's happening to a message after sending.

**Reply by thepersonthatexists** [reply]
The code is the example code replaced with my credentials which I have double checked, and there is no debug output. The command to send the mail returns no errors and works as expected. My mail server is on my own web hosting and does not store the sent emails, so I cannot look to see if they are there. I have heard mention of firewalls preventing mail from actually getting sent, but I am not sure how I would check this. It is worth noting that the old version of PHPMailer and PHP worked flawlessly, so I'm not sure what could have changed.

**Comment by Synchro** [comment]
You will only get debug output if you ask for it with `$mail->SMTPDebug = 3`. You may not be keeping whole messages, but you should have logs of transfers with any common mail server, usually somewhere like `/var/log/mail.log`. You're giving us nothing to go on here so we can only guess.
